psi4/src/psi4/libfock/cubature.cc

```diff
@@ -4560,23 +4533,6 @@ void MolecularGrid::print(std::string out, int /*print*/) const {
     Process::environment.globals["XC GRID SPHERICAL POINTS"] = options_.nangpts;
     Process::environment.globals["XC GRID RADIAL POINTS"] = options_.nradpts;
 }
-void MolecularGrid::print_details(std::string out, int /*print*/) const {
-    std::shared_ptr<psi::PsiOutStream> printer = (out == "outfile" ? outfile : std::make_shared<PsiOutStream>(out));
-    printer->Printf("   > Grid Details <\n\n");
-    for (size_t A = 0; A < radial_grids_.size(); A++) {
-        printer->Printf("    Atom: %4zu, Nrad = %6d, Alpha = %11.3E:\n", A, radial_grids_[A]->npoints(),
-                        radial_grids_[A]->alpha());
-        for (size_t R = 0; R < spherical_grids_[A].size(); R++) {
-            double Rval = radial_grids_[A]->r()[R];
-            double Wval = radial_grids_[A]->w()[R];
-            int Nsphere = spherical_grids_[A][R]->npoints();
-            int Lsphere = spherical_grids_[A][R]->order();
-            printer->Printf("    Node: %4zu, R = %11.3E, WR = %11.3E, Nsphere = %6d, Lsphere = %6d\n", R, Rval, Wval,
-                            Nsphere, Lsphere);
-        }
-    }
-    printer->Printf("\n");
-}
```

Comment:
It looks like you are removing printout of useful information. This code should be ported to use the actual data used in the code.

Reply:
The data in SphericalGrid was never correct to begin with so I'd argue it was never very important (otherwise someone would've reported the obvious printout of 0s by now). I would put good money on nobody ever actually using this print statement in anger (it showed up in v.cc, but again, SphericalGrid has been broken for >5 years and nobody reported the weird print out).

Reply:
Printing has been restored by shoving the desired data into some simple structs